controller.rst

```diff
@@ -407,20 +431,6 @@ For example, imagine you're processing a :doc:`form </forms>` submission::
         return $this->render(...);
     }
 
-.. tip::
-
-    As a developer, you might prefer not to extend the ``Controller``. To
-    use the flash message functionality, you can request the flash bag from
-    the :class:`Symfony\\Component\\HttpFoundation\\Session\\Session`::
```

Comment:
I think we need to re-read and re-think that article in its entirety... since controllers as services are a first-class citizen (does it make sense to still have a separate article?)